crowbar_framework/app/models/neutron_service.rb

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+require_relative '../helpers/barclamp/neutron_helper.rb'
```

Comment:
This should be part of the model. And the view (or even the helper which is some layer on top of the view) should access the model methods.